### ISSUE TITLE
add support for not so complex shapes (hexagon/trapezoid)

### ISSUE THREAD

**Issue by ghost** [Traewelling/line-colors#154]
just as an idea for hexagons and trapezoids:
would it be possible to support them in the near future?

we really don't need to re-invent the wheel.
i think two triangles and a rectangle might be the way to go with the existing codebase, right?

![image](https://github.com/Traewelling/line-colors/assets/160403915/27aa8715-1604-4740-8e7b-315e2e204acc)

those triangles at each end would just need to be the same colour as the rectangle.

```
                .triangle-left {
                    border-width: 50px 25px 50px 0;
                    border-color: transparent #ff0000 transparent transparent;
                }

```
`code for the already existing rectangle`
````
                .triangle-right {
                    border-width: 50px 0 50px 25px;
                    border-color: transparent transparent transparent #ff0000;
                }
````

same thing with trapezoids, which are just hexagons but in not-so-cool

![image](https://github.com/Traewelling/line-colors/assets/160403915/b76c0296-458a-41da-b37d-3016c47b17e1)

**Comment by verbuchselt** [comment]
I'm all for it.. just out of curiousity: Where do the trapezoids occur "naturally"? 😅

**Comment by lunas-bad-coding** [comment]
> I'm all for it.. just out of curiousity: Where do the trapezoids occur "naturally"? 😅

afaik: only hamburg's ferries use it.

<img width="337" alt="image" src="https://github.com/Traewelling/line-colors/assets/160403915/1056babb-29eb-45d0-9830-7bd8b50efb00">

[sauce](https://www.hvv.de/resource/blob/10938/bf77289858a97f1ec2c716b2320dd00a/hvv_linienplan_hafenfaehren_hamburg.pdf)

**Comment by CHirch** [comment]
Good evening, any updates on this? There is an example for HVV busses and ferries on the startpage of this project but there are zero hexagons in the database. Is the feature implemented or will it be implemented at some point?

**Comment by TheOneWithTheBraid** [comment]
I guess, this can be closed ; it's been implemented with #253 and https://github.com/Traewelldroid/traewelldroid/pull/465 .
